march_hardware/include/march_hardware/EtherCAT/EthercatSDO.h

```diff
@@ -6,9 +6,15 @@
 
 namespace march4cpp
 {
-int sdo_bit8(int slave, uint32_t index, uint8_t sub, uint8_t value);
-int sdo_bit16(int slave, uint32_t index, uint8_t sub, uint16_t value);
-int sdo_bit32(int slave, uint32_t index, uint8_t sub, uint32_t value);
+
+enum class SDOMessageLength
+{
+  bit8,
+  bit16,
+  bit32
+};
```

Comment:
If you give this enum a value there is no need to have a switch case

Reply:
I still need a switch statement for the value address argument (6th argument) of the ec_SDOwrite function. I need to pass different variables (uint8, uint16 or uint32) there based on the length.